alphas/alpha_functions.py

```diff
@@ -10,6 +10,7 @@
 from __future__ import annotations
 
 import pandas as pd
+from alpha_utils import *  # noqa 401,403
```

Comment:
is the `#noqa 401, 403` for flake8 or black? If it is for flake, add it to the `.flake8` file. If not, you should be able to add it to the `pyproject.toml` file under `[tool.black]`.

Reply:
I checked and they are flake8 ignore commands. Feel free to add them to the `.flake8` file. I think the letter for both is `F`. Make that edit so we can merge this PR